wrappers/python/simtk/openmm/app/internal/customgbforces.py

```diff
@@ -377,36 +393,59 @@ def _createEnergyTerms(force, solventDielectric, soluteDielectric, SA, cutoff, k
         E.sulfur : (0.96, 0.602256336067, -0.703469),
         None : (0.8, 0.5, 0.5) # default
 }
+
 _SCREEN_PARAMETERS[E.deuterium] = _SCREEN_PARAMETERS[E.hydrogen]
 def _screen_parameter(atom):
     if atom.element in _SCREEN_PARAMETERS:
         return _SCREEN_PARAMETERS[atom.element]
     return _SCREEN_PARAMETERS[None]
 
+
 class CustomAmberGBForce(CustomGBForce):
+    """Base class for all of the Amber custom GB forces.
+
+    Cannot be instantiated.
+
+    """
+    __metaclass__ = abc.ABCMeta
```

Comment:
Well, Python 3 won't *choke* on it, but it won't actually apply the metaclass.  This is what I get for Python 3:

```python
Python 3.5.1 |Continuum Analytics, Inc.| (default, Dec  7 2015, 11:24:55) 
[GCC 4.2.1 (Apple Inc. build 5577)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import abc
>>> class MyClass(object):
...     __metaclass__ = abc.ABCMeta
... 
>>> type(MyClass)
<class 'type'>
```

And for Python 2 where this syntax is used for metaclasses:

```python
Python 2.7.11 (default, May  3 2016, 22:20:22) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import abc
>>> class MyClass(object):
...     __metaclass__ = abc.ABCMeta
... 
>>> type(MyClass)
<class 'abc.ABCMeta'>
```